src/drivers/Qt/ConsoleVideoConf.cpp

```diff
@@ -748,6 +748,11 @@ void ConsoleVideoConfDialog_t::winFullScreenBorderChanged(int value)
 	g_config->setOption("SDL.winFullScreenBorder", opt );
 	g_config->save();
 }
+#elif defined(WIN32)
+void ConsoleVideoConfDialog_t::winFullScreenBorderChanged(int value)
+{
+	(void)value;
```

Comment:
The project has macros defined for handling purposely unused variables. See FCEU_UNUSED.